src/mixins/editorMixin.js

```diff
@@ -86,10 +86,34 @@ export default {
 				onOutlineToggle: (visible) => {
 					this.toggleOutlineFromEditor(visible)
 				},
+				onLoaded: () => {
+					console.log('onLoaded')
```

Comment:
With https://github.com/nextcloud/text/pull/5489 this will work. Definitely cleaner than the `setTimeout()` approach :blush:

Reply:
It is working now! :tada: